src/django/api/views/facility/facilities_view_set.py

```diff
@@ -904,9 +904,8 @@ def claim(self, request, pk=None):
 
             facility_claim.facility_workers_count = workers_count
 
-            if local_language_name != '':
-                facility_claim.\
-                    facility_name_native_language = local_language_name,
+            facility_claim.\
```

Comment:
Why did you remove this condition?

Reply:
It’s useless + I remove comma in the end that bring issue. @VadimKovalenkoSNF

Reply:
Ack